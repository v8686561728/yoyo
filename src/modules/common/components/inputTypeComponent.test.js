import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import { InputTypeComponent } from "./InputTypeComponent";

let props1 = {
  endAdornment: false,
  startAdornment: true,
  handleInputChange: jest.fn()
};
it("should render InputTypeComponent", () => {
  let wrapper = shallow(<InputTypeComponent {...props1} />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

let props2 = {
  endAdornment: true,
  startAdornment: false,
  endAdornmentIcon: true
  //   endAdornmentText: true
};
it("should render InputTypeComponent", () => {
  let wrapper = shallow(<InputTypeComponent {...props2} />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

it("should render InputTypeComponent", () => {
  let wrapper = shallow(<InputTypeComponent {...props1} />);
  shallowToJson(wrapper).children[0].children[1].props.onChange();
});