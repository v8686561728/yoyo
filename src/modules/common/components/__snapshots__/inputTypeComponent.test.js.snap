// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InputTypeComponent 1`] = `
<Fragment>
  <WithStyles(ForwardRef(FormControl))>
    <WithStyles(ForwardRef(InputLabel)) />
    <WithStyles(ForwardRef(Input))
      endAdornment={<React.Fragment />}
      onChange={[Function]}
      startAdornment={
        <ForwardRef(WithStyles)
          position="start"
        >
           
           
        </ForwardRef(WithStyles)>
      }
    />
    <WithStyles(ForwardRef(FormHelperText))>
       
       
    </WithStyles(ForwardRef(FormHelperText))>
  </WithStyles(ForwardRef(FormControl))>
</Fragment>
`;

exports[`should render InputTypeComponent 2`] = `
<Fragment>
  <WithStyles(ForwardRef(FormControl))>
    <WithStyles(ForwardRef(InputLabel)) />
    <WithStyles(ForwardRef(Input))
      endAdornment={
        <ForwardRef(WithStyles)
          position="end"
        >
           
          <ForwardRef(WithStyles)>
            true
          </ForwardRef(WithStyles)>
           
        </ForwardRef(WithStyles)>
      }
      onChange={[Function]}
      startAdornment={<React.Fragment />}
    />
    <WithStyles(ForwardRef(FormHelperText))>
       
       
    </WithStyles(ForwardRef(FormHelperText))>
  </WithStyles(ForwardRef(FormControl))>
</Fragment>
`;
